src/libambit/device_driver_ambit3.c

```diff
@@ -579,7 +579,10 @@ static int parse_log_header(libambit_sbem0102_data_t *reply_data_object, ambit3_
         data += 19;
         break;
       case 0x8a: /* gen2+gen3 fw */
-        data += 1; // Skip size ??
+        if (fw_gen == AMBIT3_FW_GEN3)
+            data += 1; // Skip size
```

Comment:
Done in pull request #2. I keep these modifications on my branch if some people just wants it to work for now.

In my dump, I only have 0x8a [size on one byte] [time] (eg : 0x8a 0x95 2017-04-29T1...) . You can see it in [monambit-openambit.zip](https://github.com/landas/openambit/files/967960/monambit-openambit.zip)

It only contains a dummy activity because I upgraded my firmware before finalizing code changes.